client/src/utils.ts

```diff
@@ -229,17 +229,18 @@ export const deepHtmlStringMatch = (element: any, match: string) => {
 
 // Given a HTML img element, process image data for barcode string
 export const processImageBarcode = async (img: HTMLImageElement) => {
-  const codeReader = new BrowserMultiFormatReader();
-  if (img) {
-    const result = await codeReader
-      .decodeFromImage(img)
-      .then((res: any) => res.text)
-      .catch((err: any) => null);
-    if (result) {
+  try {
+    const codeReader = new BrowserMultiFormatReader();
+    if (img) {
+      const result = await codeReader
+        .decodeFromImage(img)
+        .then((res: any) => res.text)
+        .catch((err: any) => null);
```

Comment:
It will automatically be nothing
```suggestion
        .catch((err: any) => {});
```

Reply:
expected return format string, changed to `""` instead